.github/workflows/workflow.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+##################################
+##    Running all unit tests    ##
+##################################
+  unitTests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Compile
+        run: |
+          cd server
+          sbt compile
+
+      - name: Test
+        run: |
+          cd server
+          sbt test
+
+# Copy logs to 1 directory, since upload-artifact doesnt support multiple directories or regular expressions
+      - name: Save logs
+        if: always()
+        run: |
+          mkdir logsArtifacts
+          find . -name logs | xargs -I LOG cp -r --parents LOG logsArtifacts/
```

Comment:
Why is this find needed?

Reply:
https://github.com/CM-Well/CM-Well/commit/2efe933e88d11dac19995268724cdbcda16d46fa/checks?check_suite_id=248257504

Reply:
https://github.com/CM-Well/CM-Well/commit/aaa57a1ccea112fddd89b214741e30ab35154697/checks?check_suite_id=248119367#step:5:6